### ISSUE TITLE
Email pop-up window

### ISSUE THREAD

**Issue by catyneis** [YoodleLLC/FDI#5]
Estimate needed for creating an email pop-up window to solve for the emails going to browsers instead of to the email platform.

**Comment by timmccoy2yoodle** [comment]
Design/layout: 1 hour
Dev: 2 hours
Testing/QA 1 hour
Revisions: 1-2 hour
Total estimate 4-6 hours

Can have this complete by Thursday 9/13

**Comment by catyneis** [comment]
Can you inform us on the fix? We'd like to be able to detail what the experience will be.

**Comment by timmccoy2yoodle** [comment]
While logged in, a user can click on an agent’s email address link in the left hand menu. Currently it is a mailto link. 

When a user clicks the email address link, a modal window will pop up, giving the user two options: email directly (via mailto) or submit message via form. 

When the user selects Email, a system email client (Mac) or email client selection window (Windows) will open and the user can send their message directly.

The user can send a message via form submission also. Since they are logged in, name and email fields are not necessary or they might be autofilled. The message needs to be filled in with no character limit and the send button needs to be clicked. After hitting SEND, the window will close.

There will also be an close (X) or cancel button in the modal.

**Comment by timmccoy2yoodle** [comment]
There are 5 tabs and each contains a mailto: link on the left hand hand side. Confirming that we need this pop up for all of them.[Uploading... Screen Shot 2018-09-10 at 5.07.33 PM.png ]

**Comment by catyneis** [comment]
Yes, but, before we are fully approved, please review the below for our meeting tomorrow:

Client is wondering if the issue is specific to client connect or if we're seeing this globally across all emails? They want to know if it's specific to their website or if everyone is having the problem. If it's every website (meaning any email on a site you have to copy and paste it into your email platform), she's not sure it's worth the extra time. If it's just us, it sounds like we might make the fix, but I need to relay back first. 

Can you confirm that the mailto is functioning as expected?

**Comment by timmccoy2yoodle** [comment]
Updated quote: 5-7, 8 max

**Comment by catyneis** [comment]
Hold on this work.
